frontend/src/employee-frontend/components/reports/VoucherServiceProviderUnit.tsx

```diff
@@ -123,56 +162,110 @@ function VoucherServiceProviderUnit() {
               }
               styles={reactSelectStyles}
             />
-          </Wrapper>
+          </FilterWrapper>
         </FilterRow>
 
-        {rows.isLoading && <Loader />}
-        {rows.isFailure && <span>{i18n.common.loadingFailed}</span>}
-        {rows.isSuccess && (
+        { report.isSuccess && report.value.locked && (
+          <LockedDate spacing='xs' alignItems='center'>
+            <FontAwesomeIcon icon={faLockAlt}/>
+            <span>
+              {`${i18n.reports.voucherServiceProviders.locked}: ${report.value.locked.format()}`}
+            </span>
+          </LockedDate>
+        )}
+
+        <HorizontalLine slim/>
+
+        {report.isLoading && <Loader />}
+        {report.isFailure && <span>{i18n.common.loadingFailed}</span>}
+        {futureSelected && <>
+          <Gap />
+          <InfoBox wide message={i18n.reports.voucherServiceProviders.filters.noFuture}/>
```

Comment:
I think instead of showing this info box we should just remove the invalid options from the drop-downs.

Reply:
thought so first too, but the valid options for months depend on year. What happens if you change year to 2021 after having selected 09/2020? Or vice versa, if you have selected 03/2021 it may feel awkward that going to 09/2020 you must select year first or September won't be available?

Reply:
Maybe ask one of our designers about it?